.github/workflows/main.yml

```diff
@@ -1,95 +1,135 @@
 name: UI Quality Check
 
 on:
-    push:
-        branches: [main]
-    pull_request:
-        branches: [main]
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
 
 jobs:
-    prettier:
-        name: Prettier
+  prettier:
+    name: Prettier
 
-        runs-on: ubuntu-latest
-        steps:
-            - name: Stop previous
-              uses: styfle/cancel-workflow-action@0.9.1
+    runs-on: ubuntu-latest
+    steps:
+      - name: Stop previous
+        uses: styfle/cancel-workflow-action@0.9.1
+
+      - name: Checkout
+        uses: actions/checkout@v2
 
-            - name: Checkout
-              uses: actions/checkout@v2
-
-            - name: Setup Node
-              uses: actions/setup-node@v2
-              with:
-                  node-version: 16
-
-            - name: Get Deps
-              uses: bahmutov/npm-install@v1
-
-            - name: Prettier...ing
-              run: npm run format
-
-    lint:
-        name: ESLint
-
-        runs-on: ubuntu-latest
-        steps:
-            - name: Stop previous
-              uses: styfle/cancel-workflow-action@0.9.1
-
-            - name: Checkout
-              uses: actions/checkout@v2
-
-            - name: Setup Node
-              uses: actions/setup-node@v2
-              with:
-                  node-version: 16
-
-            - name: Get Deps
-              uses: bahmutov/npm-install@v1
-
-            - name: Linting
-              run: npm run lint
-
-    typecheck:
-        name: TypeScript
-
-        runs-on: ubuntu-latest
-        steps:
-            - name: Stop previous
-              uses: styfle/cancel-workflow-action@0.9.1
-
-            - name: Checkout
-              uses: actions/checkout@v2
-
-            - name: Setup Node
-              uses: actions/setup-node@v2
-              with:
-                  node-version: 16
-
-            - name: Get Deps
-              uses: bahmutov/npm-install@v1
-
-            - name: Type Checking
-              run: npm run typecheck
-
-    test:
-        name: Tests
-        runs-on: ubuntu-latest
-        steps:
-            - name: Stop previous
-              uses: styfle/cancel-workflow-action@0.9.1
-
-            - name: Checkout
-              uses: actions/checkout@v2
-
-            - name: Setup Node
-              uses: actions/setup-node@v2
-              with:
-                  node-version: 16
-
-            - name: Get Deps
-              uses: bahmutov/npm-install@v1
-
-            - name: Run jest
-              run: npm run test
+      - name: Setup Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+
+      - name: Get Deps
+        uses: bahmutov/npm-install@v1
+
+      # TODO(johnny): Should this build step be verifying that the repo
+      # isn't dirty? Seems like this is just writing out formatted results
+      # which are then ignored?
+      - name: Prettier...ing
+        run: npm run format
```

Comment:
Good catch... initially the `format` job did not fix anything it just failed if they were not formatted. We'll need to update the `format` job to now try to fix stuff.